Jenkinsfile

```diff
@@ -120,14 +120,26 @@ node {
             --fix-auth \
             --create-artifacts
       """
+
+      // Run the e2e test framework
+      sh """${env.ROOT}/contrib/jenkins/run_e2e.sh \
+            --registry gcr.io/${test_project}/catalog/ \
+	    --version ${version} \
+	    --cleanup \
+	    --create-artifacts
+      """
+
+      echo 'Run succeeded.'
     } catch (Exception e) {
+      echo 'Run failed.'
       currentBuild.result = 'FAILURE'
     } finally {
-      archiveArtifacts artifacts: 'etcd-backed*.txt', fingerprint: true
-      archiveArtifacts artifacts: 'tpr-backed*.txt', fingerprint: true
```

Comment:
Nope, instead of having "etcd" and "tpr" as the prefix, they'll have "walkthrough_etcd" or "e2e_tpr". So records are being stored for all of them. If you check out the build artifacts in Jenkins you'll see what I mean, nothing's being lost.